scripts/docker-compose.yml

```diff
@@ -122,7 +134,9 @@ services:
         condition: service_started
     environment:
       - SERVER_TRACTION_URL=${SERVER_TRACTION_URL}
+      - SERVER_LOKI_URL=${SERVER_LOKI_URL}
       - FRONTEND_TENANT_PROXY_URL=${FRONTEND_TENANT_PROXY_URL}
+      - FRONTEND_LOG_STREAM_URL=${FRONTEND_LOG_STREAM_URL}
```

Comment:
These `envvars` need to be added to the chart as well

Reply:
See [chart](charts/traction/templates/ui/configmap.yaml) if its correct.